openqaoa/problems/converters.py

```diff
@@ -317,14 +323,14 @@ def linear_constraints(self, multipliers=None) -> None:
             ]:  # Inequality constraint added as a penalty with additional slack variables.
                 constraint.name = f"C{cn}"
                 if self.unbalanced:
-                    penalty = multipliers[cn] * self.inequality_to_unbalanced_penalty(
+                    penalty = self.inequality_to_unbalanced_penalty(
```

Comment:
Why is multipliers removed from this formula.

Reply:
I decided to follow the structure of the paper (https://arxiv.org/abs/2211.13914). In this case, I don't need to multiply this penalization term (which comes from the equality constraints) and the unbalanced penalization term. I split this pull request into two, the part where I covered the problems with `FromDocplex2IsingModel` are in #198.